modules/overview-of-kueue-resources.adoc

```diff
@@ -0,0 +1,136 @@
+:_module-type: CONCEPT
+
+[id='overview-of-kueue-resources_{context}']
+= Overview of Kueue resources
+
+[role='_abstract']
+The cluster administrator can configure Kueue resource flavors, cluster queues, and local queues to manage distributed workload resources across multiple nodes in an OpenShift cluster.
+
+== Resource flavour
+The Kueue `ResourceFlavor` object describes the resource variations that are available in a cluster. 
+
+Resources in a cluster can be _homogenous_ or _heterogeneous_:
+
+* Homogeneous resources are identical across the cluster: same node type, CPUs, memory, accelerators, and so on.
+* Heterogeneous resources have variations across the cluster.
+
+If a cluster has homogeneous resources, or if it is not necessary to manage separate quotas for different flavors of a resource, the cluster administrator can create an empty `ResourceFlavor` object named `default-flavor`, without any labels or taints, as follows:
+
+.Empty Kueue resource flavor for homegeneous resources
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: default-flavor
+----
+
+If a cluster has heterogeneous resources, the cluster administrator can define a different resource flavor for each variation in the resources available. 
+Example variations include different CPUs, different memory, or different accelerators.
+The cluster administrator can then associate the resource flavors with cluster nodes by using labels, taints, and tolerations, as shown in the following example.
+
+.Example Kueue resource flavor for heterogeneous resources
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "spot"
+spec:
+  nodeLabels:
+    instance-type: spot
+  nodeTaints:
+  - effect: NoSchedule
+    key: spot
+    value: "true"
+  tolerations:
+  - key: "spot-taint"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+----
+
+For more information about configuring resource flavors, see link:https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/[Resource Flavor] in the Kueue documentation.
+
+
+== Cluster queue
+
+The Kueue `ClusterQueue` object manages a pool of cluster resources such as pods, CPU, memory, and accelerators. 
+A cluster can have multiple cluster queues, and each cluster queue can reference multiple resource flavors.
+
+The cluster administrator can configure a cluster queue to define the resource flavors that the queue manages, and assign a quota for each resource in each resource flavor.
+The cluster administrator can also configure usage limits and queueing strategies to apply fair sharing rules across multiple cluster queues in the cluster.
```

Comment:
Consider omitting "in the cluster".

Reply:
@chtyler I think we need to keep the text because we don't support sharing across multiple clusters, but I changed it to "in a cluster" instead - is that any better?

@Fiona-Waters WDYT about this point in particular? Can you also please review all of the changes in the latest commit to make sure you're happy with them?

Reply:
I think we should keep it, it removes any ambiguity.

Reply:
Changes in the most recent commit look good to me.

Reply:
Looks good to me. I had suspected it was potentially redundant, but I think keeping it would suffice and would remove ambiguity.

Reply:
Thank you both :)